packages/plugin-core/src/test/suite-integ/InsertNoteIndexCommand.test.ts

```diff
@@ -39,6 +39,90 @@ suite("InsertNoteIndex", function () {
       });
     });
 
+    test("tags.foo without tags note", (done) => {
+      runLegacyMultiWorkspaceTest({
```

Comment:
This test harness is now legacy and we are slowly moving them into new ones (`describeMultiWS`). Can you use the new test harness instead? Docs here: https://docs.dendron.so/notes/veJtAvr1gSMu50Mp/

It would also be great if you can organize the tests in the GIVEN-WHEN-THEN vocabulary described here: https://docs.dendron.so/notes/42iCXsVuFhAGs3IS83KFp/

Reply:
(don't worry about the existing tests that use the old harness in this file.)

Reply:
I'm sorry for the late response.. I updated the tests to use `describeMultiWS`.

Reply:
All good 🙂 thanks for updating the tests